### PR TITLE
vert.x 4.0.0

### PR DESCRIPTION
**Pull request by SMillerDev**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by SMillerDev** [comment]
New URL comes from https://vertx.io/download/

**Comment by SMillerDev** [comment]
Not sure what the issue is with livecheck @samford ?

**Comment by samford** [comment]
The [downloads page](https://vertx.io/download/) used to have a link to the tarball but now it doesn't, so the check doesn't work anymore. The download page says that 4.x packages are on Maven Central and 3.x packages are on Bintray, so we'll have to check Maven instead.

I pushed a commit with an updated `livecheck` block that brings it up to date.

File: Formula/vert.x.rb [hunk 1]
```diff
@@ -1,13 +1,13 @@
 class VertX < Formula
   desc "Toolkit for building reactive applications on the JVM"
   homepage "https://vertx.io/"
-  url "https://bintray.com/vertx/downloads/download_file?file_path=vert.x-3.9.4-full.zip"
-  sha256 "ea81f1c64333d2372cc9a628c7a7c3491b4ba5b059558fb84137c5b2bf4ecc9e"
+  url "https://search.maven.org/remotecontent?filepath=io/vertx/vertx-stack-manager/4.0.0/vertx-stack-manager-4.0.0-full.tar.gz"
```
Comment (Rylan12) [inline Formula/vert.x.rb:4]:
```suggestion
  url "https://repo1.maven.org/maven2/io/vertx/vertx-stack-manager/4.0.0/vertx-stack-manager-4.0.0-full.tar.gz"
```

It looks like that URL redirects here (which is also the URL I see linked on the [download page](https://repo1.maven.org/maven2/io/vertx/vertx-stack-manager/4.0.0/)). Is there a reason to use the `search.maven.org` URL instead?

Comment (SMillerDev) [reply Formula/vert.x.rb:4]: Because the audit said I should 😁

Comment (Rylan12) [reply Formula/vert.x.rb:4]: Seems fair, I guess 😅

**Review by Rylan12 — APPROVED** [review]

**Comment by vietj** [comment]
@samford Bintray packages are mirror of the Maven Central binaries

**Pull request closed without merging**